lib/vmdb/loggers/formatters/container.rb

```diff
@@ -0,0 +1,47 @@
+class ContainerFormatter < Logger::Formatter
```

Comment:
Should this be something like `FluentFormatter` instead? Is this format constant across all container logging aggregators?